utils/readsb/Makefile

```diff
@@ -18,17 +18,16 @@ PKG_LICENSE_FILES:=COPYING LICENSE
 
 PKG_MAINTAINER:=Stijn Tintel <stijn@linux-ipv6.be>
 
-PKG_BUILD_DEPENDS:=!USE_GLIBC:argp-standalone
 PKG_BUILD_PARALLEL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
 define Package/readsb/default
   SECTION:=utils
   CATEGORY:=Utilities
-  DEPENDS:=+libncurses
+  DEPENDS:=+libncurses +!USE_GLIBC:argp-standalone
   TITLE:=Mode-S/ADSB/TIS decoder for various devices
-  URL:=https://github.com/Mictronic/readsb
+  URL:=https://github.com/Mictronics/readsb
```

Comment:
This change is not strictly related and I can drop it if necessary

Reply:
Indeed, you either need to drop it or preferably include the fix as separate commit.